guide/src/config.md

```diff
@@ -44,4 +44,7 @@ endpoints:
       Accept-Language: en-us
       Accept: application/json
     peak_load: 15hps
-```
\ No newline at end of file
+```
+
+## Har to Yaml Converter
+If the load test is hitting endpoints loaded by a web page, you can use the [Har to Yaml Converter](https://beta.familysearch.org/status/performance-test/yamlwriter). First you need to create a [Har File](https://docs.microsoft.com/en-us/azure/azure-portal/capture-browser-trace) from the page load, then use the Converter to generate a Yaml Config file.
```

Comment:
I might start with something like: "If you are attempting to load test a specific web page" but otherwise it works. It really depends on if we think someone is going to use this that isn't really familiar with endpoints?